lizard_neerslagradar/layers.py

```diff
@@ -102,17 +101,18 @@ def search(self, google_x, google_y, radius=None):
         name = self._grid_name(region.name, pixel)
 
         return [{
-                'distance': 0,
-                'name': name,
-                'shortname': name,
-                'workspace_item': self.workspace_item,
-                'identifier': {
-                    'identifier': pixel,
-                    'region_name': region.name
-                    },
+            'distance': 0,
+            'name': name,
+            'shortname': name,
+            'workspace_item': self.workspace_item,
+            'identifier': {
+                'identifier': pixel,
+                'region_name': region.name,
                 'google_coords': (google_x, google_y),
-                'object': None
-                }]
+            },
+            'google_coords': (google_x, google_y),
```

Comment:
Je geeft nu een keer extra google_coords door (en op een ander niveau in de dict).